src/automationClient.ts

```diff
@@ -94,12 +94,12 @@ export class AutomationClient {
 
         setLogLevel(this.configuration.logging.level);
 
-        if (this.configuration.logging.file !== false) {
+        if (this.configuration.logging.file && this.configuration.logging.file.enabled === true) {
```

Comment:
Is the first check necessary given the default configuration?

Reply:
yeah, I guess not. that's right. I'll remove it for clarity.